test/Microsoft.NET.Build.Tests/GivenThatWeWantToReferenceAnAssembly.cs

```diff
@@ -26,6 +26,13 @@ public class GivenThatWeWantToReferenceAnAssembly : SdkTest
             string referencerTarget,
             string dependencyTarget)
         {
+            if (UsingFullFrameworkMSBuild)
```

Comment:
Previously our .NET Core 2 tests were specifying a RuntimeFrameworkVersion.  I didn't want to do that for the tests I added, so I updated to a new version of the CLI where you get the bundled version of .NET Core automatically.  However, the version of full Framework MSBuild on the CLI doesn't have the props file to set the bundled framework version yet, so I had to disable some tests from running on full framework.